kmod/src/Makefile.kernelcompat

```diff
@@ -43,3 +43,15 @@ endif
 ifneq (,$(shell grep 'posix_acl_valid.*user_ns,' include/linux/posix_acl.h))
 ccflags-y += -DKC_POSIX_ACL_VALID_USER_NS
 endif
+
+#
+# rhel8+ has a nonstandard RB_DECLARE_CALLBACKS
```

Comment:
It certainly looks like el8 and upstream both do have this evolution of the rb aug callback generating macros.  Maybe we should chat more about this history here, I wonder if I'm missing something.